cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -117,7 +117,9 @@ func (i *importer) cleanupDisk() {
 }
 
 func (i importer) buildLoggable() service.Loggable {
-	return service.SingleImageImportLoggable(i.pd.sourceType, i.pd.sourceGb, i.pd.sizeGb, i.traceLogs)
+	return service.SingleImageImportLoggable(i.pd.sourceType, i.pd.sourceGb, i.pd.sizeGb,
+		i.pd.matchResult, int64(i.pd.inflationTime.Seconds()),
+		int64(i.pd.shadowInflationTime.Seconds()), i.traceLogs)
```

Comment:
Do you envision any of these metrics living beyond your test phase?

Reply:
No, I will delete them. Just filed  b/161177634 to track.